src/components/TopBanner/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Block } from '../Block/index'
+import { Icon } from '../Icon/index'
+import { Text } from '../Typography/index'
+import './topbanner.scss'
+
+export const BannerType = ['warning', 'danger', 'success', 'default']
+export const TopBanner = ({
+  type = 'default',
+  content,
+  buttons,
+  loading,
+  visible,
+  loadingText = 'Loading',
+}) => {
+  return visible ? (
+    <Block
+      className={`top-banner ${type}`}
+      display="flex"
+      justifyContent="center"
+      alignItems="center"
+    >
+      {loading && (
+        <Block>
+          <Icon.Loading size={16} color="white" />
```

Comment:
Icon.Loading has `strokeWidth` and `trackColor` props. You can use it if you want. Not sure if color="white" is what we want.